### DISCUSSION TITLE
How to modify Business Name/ Business Address on Disk Erasure Report?

### DISCUSSION THREAD

**Discussion by BigSteve50** [PartialVolume/shredos.x86_64#267] [Q&A] [answered]
Hi,

Firstly I'd like to say thank you for all the hard work that's been put into this amazing project. I've been using ShredOS for a few months now and its absolutely perfect for the wiping jobs I've needed to carry out.

I did originally change the Business Name and Address etc at the top of the report but I've noticed I've made a typo in there and would like to change it now. However, I cant for the life of me find out where you do this!? I think I'm going mad as I'm sure it was on the info page originally so maybe I'm going blind so if anyone can tell me the secret again I would be extremely grateful.

I know almost nothing about Linux and just  made a USB stick from you guide using Rufus and booting to that to wipe old machines.

Thanks!

**Comment by PartialVolume** [comment, marked as answer]
Thanks, glad it's useful.

From the drive selection screen, press c for config.

**Comment by BigSteve50** [comment]
Ah brilliant - Thanks!
